_posts/2021-01-04-proxmox-server.md

```diff
@@ -0,0 +1,249 @@
+    You can download the latest version of Proxmox directly [at this page](https://www.proxmox.com/en/downloads/category/iso-images-pve).
+2. Creating the USB key
+    I created the key on macOS Cantalina. For other OSes, you can find the instructions at the [Proxmox wiki](https://pve.proxmox.com/wiki/Prepare_Installation_Media).
+
+    ``` bash
+    hdiutil convert -format UDRW -o proxmox-ve_*.dmg proxmox-ve_*.iso # convert the proxmox image into proper format
+    ```
+    After converting the image file, plug the USB key into the Mac and looking for the disk.
+    ``` bash
+    # look for the external disk
+    diskutil list # list all the disk attached to the computer
+    ```
+    From the list of disks, there should be an external disk. We need to unmount it prior to writing the installation media.
+    ``` bash
+    diskutil unmountDisk /dev/diskX # replace X with the number corresponding to the external disk
+    ```
+    Now, you can create the install media using [dd](https://en.wikipedia.org/wiki/Dd_(Unix))
+    ``` bash
+    sudo dd if=proxmox-ve_*.dmg of=/dev/rdiskX bs=1m
+    ```
+## Installing Proxmox
+1. Installing Proxmox
+   
+   Aside from a making a couple of choices on the file system and the address of the home lab, installation of Proxmox is straight forward.
+
+2. Logging into Proxomox's Web Interface
+   
+   At this point, you can unplug the home lab box's monitor and keyboard. The rest of the guide can be done via a terminal or Proxmox's web interface.
+    ```
+    https://address.you.chose:8006
+    ```
+    Port 8006 is the default port number if you did not change it during the installation process. On visiting the web interface from a browser, the server would prompt for user name and password. The default user name is root and the password would have been chosen during the install. 
+    If the page fails to load, you may have forgotten the `s` in `https`. Because the `s` is needed, your browser may give you are warning about the webpage being insecure. You may safely ignore it. 
+
+## Configuring Proxomox for GPU Pass Through
+
+Making GPU pass through work on Proxmox and containers is essentially a two step process:
+    
+    1. configure the drivers on the server, and
+    2. configure the containers. 
+
+1. Configure the Nvidia Drivers on the Server
+
+    Either through Promox web interface or login to the server directly via SSH, we need to have command line access to the server. Since Proxmox is based on Debian, we follow the steps outlined in the [Debian wiki](https://wiki.debian.org/NvidiaGraphicsDrivers) to install the drivers on Proxmox. 
+
+    We need to add the following lines to `/etc/apt/sources.list`:
+    ``` bash
+    # security updates
+    deb http://security.debian.org buster/updates main contrib
+    
+    # PVE pve-no-subscription repository provided by proxmox.com,
+    # NOT recommended for production use
+    deb http://download.proxmox.com/debian buster pve-no-subscription
```

Comment:
This is not in Debian wiki. what does it do. why not recommended for production.

Reply:
not recommended for production because it's on the "no-subscription" branch. added proxmox wiki for clairification